python/ray/dashboard/subprocesses/handle.py

```diff
@@ -37,14 +27,41 @@
 logger = logging.getLogger(__name__)
 
 
+def filter_headers(
```

Comment:
What's the reason for having this code?

Reply:
By definition, hop-by-hop headers are meaningful only for a single transport-level connection, and must not be retransmitted by proxies.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers#hop-by-hop_headers

Reply:
filter_hop_by_hop_headers